__tests__/backend/taxjarApi.test.ts

```diff
@@ -1,39 +1,33 @@
-import _default from "taxjar/dist/util/request";
-import { createOrderTransaction, fetchTaxes } from "../../backend/taxjarApi";
 import { TaxJarConfig } from "../../backend/types";
-import { getTaxJarConfig } from "../../backend/utils";
 import {
   dummyFetchTaxesPayload,
   dummyFetchTaxesResponse,
   dummyOrderCreatedPayload,
   dummyTaxJarConfig,
 } from "../utils";
-import { TaxForOrderRes } from "taxjar/dist/util/types";
-
-jest.mock("taxjar/dist/util/request");
-
-const mockedTaxJarRequest = <jest.Mock>_default;
 
 describe("TaxJar API handlers", () => {
-  afterEach(() => {
-    jest.clearAllMocks();
+  beforeEach(() => {
+    jest.resetModules();
   });
 
   it("fetches taxes from taxjar", async () => {
-    const mockedTaxJarResponseData = dummyFetchTaxesResponse;
     const post = jest.fn((url: any, params: any) => ({
       mockedTaxJarResponseData,
     }));
-    const mockedTaxJarResponse = mockedTaxJarRequest.mockImplementationOnce(
-      () => ({
-        post: post,
-      })
-    );
+
+    jest.doMock("taxjar/dist/util/request", () => ({
+      __esModule: true,
+      default: jest.fn((_) => ({ post: post })),
+    }));
+    const taxJarApi = require("../../backend/taxjarApi");
```

Comment:
I think since you're mocking `taxjar/request` in every test case, you could just use `jest.mock` at the top of the file:

```js
jest.mock("taxjar/dist/util/request", () => ({
  __esModule: true,
  default: jest.fn((_) => ({ post: post }))
}))
```

and in the tests:

```ts
afterEach(() => {
  jest.clearAllMocks();
})
```